### ISSUE TITLE
``v doc sdl`` error

### ISSUE THREAD

**Issue by Cons-Cat** [vlang/sdl#6]
Running V doc on this module causes this problem for me:
```v
$ v doc sdl
.vmodules/sdl/sdl.v:98:29: error: unexpected name `byte`, expecting `,`
   96 | 
   97 | ///////////////////////////////////////////////////
   98 | fn C.SDL_MapRGB(fmt voidptr byte, g byte, b byte) u32
      |                             ~~~~
   99 | fn C.SDL_CreateRGBSurface(flags u32, width int, height int, depth int, Rmask u32, Gmask u32, Bmask u32, Amask u32) voidptr
  100 | fn C.SDL_PollEvent(&SDL_Event) int
```

**Comment by NrdyBhu1** [comment]
@Cons-Cat , did you find any solution?

**Comment by Larpon** [comment]
Fixed in #7

**Comment by Larpon** [comment]
@JalonSolov / @medvednikov I can't reproduce this after #7
